context/app/static/js/components/detailPage/ProcessedData/HelperPanel/HelperPanel.tsx

```diff
@@ -80,7 +80,9 @@ function HelperPanelBody() {
           {currentDataset.description}
         </HelperPanelBodyItem>
       )}
-      <HelperPanelBodyItem label="Pipeline">{currentDataset.pipeline}</HelperPanelBodyItem>
+      <HelperPanelBodyItem label="Pipeline">
```

Comment:
I think you need to update the label as well?

Reply:
Are we using different terminology for the EPIC processing types?

Reply:
@tsliaw what should the label be for EPIC type?

Reply:
I'm not sure, pipeline doesn't seem completely accurate. Maybe `Processing Type` `Externally Processing Type` something like that, but I'm really not sure. @lchoy Do you have any insights for what this label should be?

Reply:
My suggestion is "Analysis" or "Analysis Type" so as to work for all cases - those datasets that underwent pipeline processing and epics that are "transformed".

I had added a comment in the Jira: https://hms-dbmi.atlassian.net/browse/CAT-959?focusedCommentId=60307, before I came across this thread here.

Reply:
The Processed Data Summary section should be updated too, as it lists "Pipelines" and is showing "unknown" for epics.

Reply:
Updated language:
![Screenshot 2024-10-24 at 10 46 37 AM](https://github.com/user-attachments/assets/bdbf8828-6f19-4302-887b-4c368080da70)
![Screenshot 2024-10-24 at 10 47 06 AM](https://github.com/user-attachments/assets/b9d69b18-ed2a-47e9-ad6f-b71969f3af56)